libs/design/src/molecules/card/card/card.component.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../../scss/daff-util';
+
+.daff-card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  max-width: 600px;
```

Comment:
I don't think we should be enforcing a max-width.

Reply:
Resolved in 22430c3